app/ControllerFunctions/Update/Check.php

```diff
@@ -106,23 +132,38 @@ public function getText()
 	 * 1 - Not in cache
 	 * 1 - Up to date
 	 * 2 - Not up to date.
+	 * 3 - Require migration.
 	 */
 	public function getCode()
 	{
-		$update = $this->canUpdateBool();
+		if ($this->lycheeVersion->isRelease) {
+			// @codeCoverageIgnoreStart
+			$versions = $this->lycheeVersion->get();
+
+			return 3 * intval($versions['DB']['version'] < $versions['Lychee']['version']);
+		// @codeCoverageIgnoreEnd
+		} else {
```

Comment:
It's not a big deal, but we shouldn't need an `else` if there's a return. I'm only mentioning it because there are a lot of layers here.